metal/loadbalancers.go

```diff
@@ -49,7 +49,7 @@ type loadBalancers struct {
 	eipTag                string
 }
 
-func newLoadBalancers(client *packngo.Client, k8sclient kubernetes.Interface, projectID, metro, facility, config string, localASN int, bgpPass, annotationNetwork, annotationLocalASN, annotationPeerASN, annotationPeerIP, annotationSrcIP, annotationBgpPass, eipMetroAnnotation, eipFacilityAnnotation, nodeSelector, eipTag string) (*loadBalancers, error) {
+func newLoadBalancers(client *packngo.Client, k8sclient kubernetes.Interface, projectID, metro, facility, config string, localASN int, bgpPass, annotationNetwork, annotationLocalASN, annotationPeerASN, annotationPeerIP, annotationSrcIP, annotationBgpPass, eipMetroAnnotation, eipFacilityAnnotation, nodeSelector, eipTag string, useCRDForMetalLB bool) (*loadBalancers, error) {
```

Comment:
I don't love how this one flag for a specific upstream change in one load balancer implementation bubbles all the way up through each layer, but maybe that's a weird thing to focus on.